test/functional/ui/highlight_spec.lua

```diff
@@ -540,6 +540,27 @@ describe("'listchars' highlight", function()
     ]])
   end)
 
+  it("'cursorline' and 'standout'", function()
+    screen:set_default_attr_ids({
+        [1] = {bold = true, foreground = Screen.colors.Blue1},
+        [2] = {standout = true, bold = true, underline = true,
+        background = Screen.colors.Gray90, foreground = Screen.colors.Blue1},
+        [3] = {standout = true, underline = true,
+        background = Screen.colors.Gray90}
+    })
+    feed_command('hi CursorLine cterm=standout,underline term=standout,underline gui=standout,underline')
```

Comment:
`term=` can be removed, only `cterm` and `gui` should be significant.

Reply:
OK, yeah, got it